packages/deployer/CHANGELOG.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "0.4.2",
+        "changes": [
+            {
+                "note": "Add support for solidity 0.4.23",
+                "pr": 545
+            }
+        ],
+        "timestamp": 1524044013
```

Comment:
i believe the timestamp is auto generated  on publish so we should not manually add it